website_event_compassion/controllers/events_controller.py

```diff
@@ -223,16 +223,18 @@ def donation_payment_validate(self, invoice_id=None, **kwargs):
         except ValueError:
             transaction = request.env["payment.transaction"]
 
+        invoice_lines = invoice.invoice_line_ids
+        event = invoice_lines.mapped("event_id")
```

Comment:
```suggestion
        event = invoice.invoice_line_ids.mapped("event_id")
```

Reply:
we need invoice_lines after to get the ambassador so I think this should stay the way it is